.github/workflows/build-documentation.yml

```diff
@@ -37,4 +37,5 @@ jobs:
         sudo apt-get install pandoc doxygen
     - name: Build docs
       run: |
-        sphinx-build -vvv -T -E -b html -d docs/build/doctrees ./docs docs/build/html
```

Comment:
Out of interest why does this need to go? Is it identical to make html?

Reply:
as far as I can tell, yes. So having it go allows to make clear what the developer needs to run locally to build a local version of the doc while working on it.